torch_xla/experimental/fori_loop.py

```diff
@@ -12,40 +12,70 @@
 from torch._higher_order_ops.while_loop import while_loop_op
 
 
+def fori_loop(lower, upper, body_fun, one_value, init_val):
```

Comment:
nit. rename `body_fun` to `user_body_func`, to be more distinct from `body_fn` below

Reply:
Thanks, updated